tests/mgr/test_restful_operations_pool_get.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+from api.restful.pool import Pool
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError
+from utility.log import Log
+
+LOG = Log(__name__)
+
+
+# Request header
+HEADER = {
+    "Accept": "application/vnd.ceph.api.v1.0+json",
+    "Content-Type": "application/json",
+}
+
+# Missing config parameter err msg
+CONFIG_ERR_MSG = "Mandatory config parameter '{}' not provided"
+
+
+# Mandatory test config exception
+class ConfigNotFoundError(Exception):
```

Comment:
Add `ConfigError` exception under https://github.com/red-hat-storage/cephci/blob/master/api/exceptions.py

Reply:
https://github.com/red-hat-storage/cephci/blob/master/api/exceptions.py is giving 404

Reply:
create new file and add exception

Reply:
Done